db/schema.sql

```diff
@@ -30,7 +30,8 @@ create table users (
     is_superuser    boolean not null default 'f',
     title           text,
     twitter_id      bigint  unique,
-    facebook_id     bigint  unique
+    facebook_id     bigint  unique,
+    area_id         integer
```

Comment:
Discussed briefly in IRC, whether we do this, or add an extra column as with other tables. I guess an extra column is more extensible if we have other things in future, question is (given our current postgres version) do we need to query by area ID, or only look it up by user.